.gitignore

```diff
@@ -0,0 +1 @@
+vendir.lock.yml
```

Comment:
Can you elaborate why you think this file shouln't be versioned?

Reply:
Well, for me it's the other way around: Why should you version it? I don't see any value nor use case in it. 🤷

Reply:
The file is created by vendir to make the vendir sync reproducible and also give context to which git sha/whatever it synced from. IMO having this information in PRs is worth keeping the file in the repository and versioned.

Reply:
Ok, you're right. It's fine for keeping track of from which ref we synced.

Reply:
Thanks for keeping it :)